_source/_posts/2024-12-30-scim-okta-integration-network.md

```diff
@@ -0,0 +1,72 @@
+---
+layout: blog_post
+title: "How to Submit Your System for Cross-domain Identity Management (SCIM) Integration to the Okta Integration Network"
+author: semona-igama
+by: advocate
+communities: [devops,security,mobile,.net,java,javascript,go,php,python,ruby]
+description: "A post on creating a SCIM provisioning instance in Okta for testing and submitting to the Okta Integration Network."
```

Comment:
No need to add "Ad post on..." Description for what this post is about. You're trying to entice people to click on the link